sklearn/utils/__init__.py

```diff
@@ -195,15 +197,16 @@ def resample(*arrays, **options):
     if max_n_samples is None:
         max_n_samples = n_samples
 
-    if max_n_samples > n_samples:
```

Comment:
A nitpick that does not really matter.

I would write `if max_n_samples > n_samples and not replace` instead of the nested if below